packages/angular/src/generators/mfe-remote/mfe-remote.ts

```diff
@@ -1,7 +1,23 @@
 import type { Tree } from '@nrwl/devkit';
 import type { Schema } from './schema';
-import { getProjects } from '@nrwl/devkit';
+import { getProjects, readProjectConfiguration } from '@nrwl/devkit';
 import applicationGenerator from '../application/application';
+import { getMfeProjects } from '../../utils/get-mfe-projects';
+
+function findNextAvailablePort(tree: Tree) {
+  const mfeProjects = getMfeProjects(tree);
+
+  const ports = new Set<number>();
+  for (const mfeProject of mfeProjects) {
+    const { targets } = readProjectConfiguration(tree, mfeProject);
+    const port = targets?.serve?.options?.port ?? 4200;
+    ports.add(port);
+  }
+
+  const nextAvailablePort = [...ports].sort((a, b) => b - a)[0] + 1;
```

Comment:
nit: we could improve readability a bit by doing:
```suggestion
  const nextAvailablePort = Math.max(...ports) + 1;
```

Reply:
I like that a lot!